src/test/java/HttpEventCollector_JavaLoggingTest.java

```diff
@@ -271,9 +283,8 @@ public void error(final List<HttpEventCollectorEventInfo> data, final Exception
             Thread.sleep(1000);
         }
 
-        if (logEx == null)
-            Assert.fail("didn't catch errors");
-
+        Assert.assertNotNull(logEx);
+        Assert.assertEquals(1, logEx.size());
```

Comment:
This test is pretty unreliable for me locally. I'll just revert changes to this file since they are unrelated to the change.